standalone-metastore/metastore-server/src/test/java/org/apache/hadoop/hive/metastore/TestRemoteHiveMetastoreWithHttpJwt.java

```diff
@@ -102,9 +104,10 @@ public static void makeEnvModifiable() throws Exception {
 
   private static void removeStaticFinalAndSetValue(Field field, Object value) throws Exception {
     field.setAccessible(true);
-    Field modifiersField = Field.class.getDeclaredField("modifiers");
-    modifiersField.setAccessible(true);
-    modifiersField.setInt(field, field.getModifiers() & ~Modifier.FINAL);
+    VarHandle modifiersHandle = MethodHandles.privateLookupIn(Field.class, MethodHandles.lookup())
```

Comment:
move to common place, ReflectionUtils

Reply:
this has been addressed in [3402fe5](https://github.com/apache/hive/pull/5404/commits/3402fe5662ae7db2362f805e6591e1d672bcd1fc)